lib/src/json_serializable_generator.dart

```diff
@@ -76,7 +76,7 @@ class JsonSerializableGenerator
     // Explicitly using `LinkedHashMap` – we want these ordered.
     var fields = new LinkedHashMap<String, FieldElement>.fromIterable(
         fieldsList,
-        key: (f) => f.name);
+        key: (FieldElement f) => f.name);
```

Comment:
[aside] I'm a little surprised this one is necessary - is fieldsList not well typed?